src/chatgptClient.ts

```diff
@@ -23,6 +23,16 @@ export async function getSuggestions(
 
```

Comment:
### Line 23
## CodeGuard Suggestions
**Suggestion:** Extract the json from the response with usual string parsing techniques
**Reason:** This process of extracting json from response data is easier and more efficient than using the methods in this block